pages/pipelines/rules.md

```diff
@@ -0,0 +1,169 @@
+
+## Pipeline rules best practices
+
+### How should I use rules?
+
+Use rules to manage permissions between pipelines and other resources. Rules are typically used in tandem with [clusters](/docs/clusters/overview) to increase security and control, where clusters set hard boundaries and rules provide exceptions.
+
+### Available rules
+
+#### `pipeline.trigger_build.pipeline` 
+
+Cross-cluster pipeline triggering (eg. allow a pipeline in one cluster to trigger a pipeline in another cluster).
+
+Value fields:
+
+- `triggering_pipeline_uuid` The UUID of the pipeline that is allowed to trigger another pipeline.
+- `triggered_pipeline_uuid` The UUID of the pipeline that is allowed to be triggered by the `triggering_pipeline_uuid` pipeline.
+
+### `pipeline.artifacts_read.pipeline` Allowing jobs in a pipeline to access artifacts from builds in other pipelines.
+
+Value fields:
+
+- `source_pipeline_uuid` The UUID of the pipeline that is allowed to access artifacts from another pipeline.
+- `target_pipeline_uuid` The UUID of the pipeline that is allowed to have its artifacts accessed by jobs in the `source_pipeline_uuid` pipeline.
+
+### Example rule use
+
+Imagine you use two clusters to separate environments necessary for building your application (CI cluster) and deploying your application (CD cluster). Ordinarily pipelines in those clusters would not be able to trigger each other due to the isolation of clusters.
+
+Creating a `pipeline.trigger_build.pipeline` rule would allow triggering a pipeline in the CD cluster from a pipeline in the CD cluster. This would allow maintaining the separation of the CI and CD agents and still support triggering the deployment pipeline from the build pipeline.
+
+An example of a rule that allows a pipeline in the CI cluster to trigger a pipeline in the CD cluster:
+
+```json
+{
+  "rule": "pipeline.trigger_build.pipeline",
+  "value": {
+    "triggering_pipeline_uuid": "{uuid-of-build-pipeline}",
+    "triggered_pipeline_uuid": "{uuid-of-deploy-pipeline}"
+  }
+}
+```
+
+## Create a rule
+
+Admins can create new rules on the **Rules** page in the **Organization settings**, as well as through the REST API's or GraphQL API's create-a-rule feature.
+
+### Using the Buildkite interface
+
+To create a new cluster using the Buildkite interface:
+
+1. Select **Settings** in the global navigation to access the **Organization settings** page.
+2. Select the **Rules** page in the Pipelines section.
+3. Select **New rule**.
+4. Under **Rule name**, select the type of rule you want to create, such as `pipeline.trigger_build.pipeline`.
+5. Fill out the UUIDs of the pipelines you wish to create a target for. Access the UUIDs of your pipelines on their **Settings** page under the **GraphQL API integration** section.
+6. Select **Submit**.
+
+### Using the REST API
```

Comment:
These two bottom API sections feel like they belong in the dedicated Buildkite REST and GraphQL API sections of the docs? Maybe we could shift the detail over there, and just link to it from here